test/val/val_barriers_test.cpp

```diff
@@ -251,7 +253,7 @@ OpControlBarrier %workgroup %workgroup %acquire_release_uniform_workgroup
 
 TEST_F(ValidateBarriers, OpControlBarrierWebGPUAcquireReleaseSuccess) {
   const std::string body = R"(
-OpControlBarrier %workgroup %workgroup %acquire_release_uniform_workgroup
+OpControlBarrier %workgroup %workgroup %acquire_release_workgroup
```

Comment:
I think you should duplicate this test as a failure for including uniform

Reply:
Apparently I moved the Uniform test lower down, and removed a different test :-S

I have added back in the missing test, and the test you asked for should be at around 286